lib/ui/home/home_screen.dart

```diff
@@ -42,16 +80,59 @@ class _HomeScreenState extends ConsumerState<HomeScreen> {
               },
           orElse: () => {});
     });
+
     return Scaffold(
-      body: SafeArea(
-        child: Consumer(
-          builder: (context, widgetRef, _) {
-            return HomeHeader(
-              date: DateTime.now().getFormattedString(),
-              avatar: widgetRef.watch(_profileAvatarProvider),
-            );
-          },
-        ),
+      body: Stack(
+        children: [
+          PageView.builder(
+            controller: _pageController,
+            onPageChanged: (index) {
+              setState(() {
+                _currentIndex = index;
+              });
+            },
+            itemCount: mockSurveys.length,
+            itemBuilder: (_, index) {
+              return HomeSurveyItem(
+                survey: mockSurveys[index],
+                onNextButtonPressed: () {
+                  // TODO: Navigate to detail screen
+                  showToastMessage(_currentIndex.toString());
+                },
+              );
+            },
+          ),
+          SafeArea(
+            child: Consumer(
+              builder: (context, widgetRef, _) {
+                return HomeHeader(
+                  date: DateTime.now().getFormattedString(),
```

Comment:
@Wadeewee please add a TODO comment to move this to ViewModel later 😄 the logic to get current day of the survey should be from VM side

Reply:
Added @manh-t 🚀